pkg/services/alerting/rule_test.go

```diff
@@ -60,7 +60,7 @@ func TestAlertRuleModel(t *testing.T) {
 		})
 
 		Convey("can construct alert rule model", func() {
```

Comment:
Can you add a test for trying to create an alert rule with invalid (not-existing) notification id?

Reply:
with pleasure :)